### DISCUSSION TITLE
Smart Search results shows tag but not "tagged" article

### DISCUSSION THREAD

**Discussion by RustyIngles** [joomla/joomla-cms#26545] [2. Cold Features]
### Steps to reproduce the issue

New install of Joomla 3.9.12
Create a new article, call it "Test Article" and add a tag of "october"
Index the Content (making sure all Smart Search plugins are enabled beforehand)
In the frontend, search for "test article" the article is shown in the results.
In the frontend, search for "october", it shows the tag being returned, but it doesn't show the link to the article within the results. It only shows a link to the tag, which when clicked takes you through to a list of the article(s) with the tag

### Expected result

On first search, the article title that has the tag should appear in the search results, not a link to all articles with the tag

### Actual result

It shows the tag being returned, but it doesn't show the link to the article within the results. It only shows a link to the tag, which when clicked takes you through to a list of the article(s) with the tag

### System information (as much as possible)

Database Type | mysql
Database Version | 8.0.16
Database Collation | utf8_general_ci
Database Connection Collation | utf8mb4_0900_ai_ci
PHP Version | 7.3.3
Web Server | Apache
WebServer to PHP Interface | apache2handler
Joomla! Version | Joomla! 3.9.12 Stable [ Amani ] 24-September-2019 15:00 GMT
Joomla! Platform Version | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36

### Additional comments

I've tested this on several sites and they're all showing the same result

**Comment by SharkyKZ** [comment]
Current behavior is correct. Tags could probably be used as taxonomy data to get your expected result.
